protocol/v2/qbft/storage/participant_store.go

```diff
@@ -7,9 +7,21 @@ import (
 
 	specqbft "github.com/ssvlabs/ssv-spec/qbft"
 	spectypes "github.com/ssvlabs/ssv-spec/types"
-	"github.com/ssvlabs/ssv/exporter/convert"
 )
 
+type Participation struct {
+	ParticipantsRangeEntry
+
+	DomainType spectypes.DomainType
+	Role       spectypes.BeaconRole
+	PK         spectypes.ValidatorPK
+}
+
+func (p *Participation) MsgID() spectypes.MessageID {
+	// nolint: gosec
+	return spectypes.NewMsgID(p.DomainType, p.PK[:], spectypes.RunnerRole(p.Role))
+}
```

Comment:
i forgot the new MessageID has a different ordering than the old one (i think role and pubkey switched places)

in order to behave backwards-compatibility for exporter users, i think that instead, we should copy the content of the NewMessageID/NewMsgID function from the genesis spec into this func here, but return the bytes array directly rather than the MessageID type

you can find the correct genesis spec repo & commit by following this import in go.mod:

```
github.com/ssvlabs/ssv-spec-pre-cc v0.0.0-20240725052506-c48532da6a63
```

(pre-cc just means pre-committee-consensus which i know is a weird name but going to be deleted soon)

plus, i think we should rename MsgID method here to legacyMsgID just to make it clear its for API backwards-compatibility and not for other usage (a comment would also help)